assets/src/components/form/validations/index.js

```diff
@@ -0,0 +1,194 @@
+/**
+ * External imports
+ */
+import { __, _n, sprintf } from '@eventespresso/i18n';
+
+/**
+ * @function
+ * @param {mixed} value
+ * @return {string|null} error string on fail
+ */
+export const isFloat = value => {
+	// first convert to string
+	value = value + '';
+	// convert decimal to known format
+	value = value.replace( ',', '.' );
+	// strip all trailing zeros
+	value = value.replace( /0+$/, '' );
+	const lastChar = value.substr( value.length - 1 );
+	const float = (
+		// parsed value is same as original coerced to a number
+		parseFloat( value, 10 ) === +value &&
+		(
+			// no remainder but last character is a decimal
+			// ie: original value like 1.0
+			( value % 1 === 0 && lastChar === '.' ) ||
+			// has a remainder and last character is NOT a decimal
+			// ie: original value like 1.# where # is not 0
+			( value % 1 !== 0 && lastChar !== '.' )
+		)
+	);
+	return ! float ? __( 'Value must be a float', 'event_espresso' ) : null;
+};
+
+/**
+ * @function
+ * @param {mixed} value
+ * @return {string|null} error string on fail
+ */
+export const isInteger = value => {
+	return ! ( parseInt( value, 10 ) === +value || value % 1 === 0 ) ?
+		__( 'Value must be an integer', 'event_espresso' ) :
+		null;
+};
```

Comment:
Just a fyi regarding general ES6 syntax, this could have been written as:

```
export const isInteger = value => ! ( parseInt( value, 10 ) === +value || value % 1 === 0 ) ?
		__( 'Value must be an integer', 'event_espresso' ) :
		null;
```
Slightly more condensed.  The way you did it is fine, but its nice to eliminate extra brackets etc when possible.

Reply:
kk will do 👍